docs/whats-new.asciidoc

```diff
@@ -4,131 +4,93 @@
-
-The {security-guide}/rule-monitoring-dashboard.html[Detection rule monitoring dashboard] provides visualizations to help you monitor the overall health and performance of {elastic-sec}'s detection rules. Review this dashboard for a high-level overview to determine if your rules are running successfully and how long they’re taking to run, search data, and create alerts.
-
-[role="screenshot"]
-image::whats-new/images/8.9/rule-monitor-dashboard.png[Detection rule monitor dashboard]
+== Detection rules and alerts enhancements
 
 [float]
-=== Automated endpoint response actions for rules
-
-You can now add the {security-guide}/host-isolation-ov.html#isolate-a-host[host isolation response action] to rules. When rule conditions are met, the endpoint is automatically isolated. 
+=== MITRE ATT&CK® coverage page
 
-[float]
-=== Rule exceptions auto-populated with alert data
+The {security-guide}/rules-coverage.html[MITRE ATT&CK® coverage page] shows which MITRE ATT&CK® adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.
 
-Now, when you {security-guide}/add-exceptions.html#detection-rule-exceptions[create a new rule exception] from an alert, exception conditions are auto-populated with relevant alert data. A comment describing this action is also automatically added to the *Add comments* section.
+[role="screenshot"]
+image::whats-new/images/8.10/rules-coverage.png[MITRE ATT&CK® coverage page]
 
 [float]
-=== Interactive investigation guides are now generally available
-
-{security-guide}/interactive-investigation-guides.html[Interactive investigation guides], which suggest steps for triaging, analyzing, and responding to potential security issues, are now generally available. You can configure an interactive investigation guide when you create a new rule or edit an existing one.
+=== New prebuilt rule details flyout
 
+The new prebuilt rule details flyout allows you to examine the details of a prebuilt rule before you {security-guide}/prebuilt-rules-management.html[install or update] it. You can access this flyout by clicking a rule name on either the **Add Elastic Rules** page or the **Rule updates** table. The flyout displays the About, Definition, and Schedule sections, as shown on the rule details page. It also shows the setup and investigation guides for rules that have them.
+ 
 [role="screenshot"]
-image::whats-new/images/8.9/IG-UI.png[Interactive investigation guide]
-
+image::whats-new/images/8.10/prebuilt-rule-details-flyout.png[Prebuilt rule details flyout]
 
 [float]
-=== Prebuilt rule updates
+=== Redesigned alert details flyout
 
-Check out the {security-guide}/prebuilt-rules-downloadable-updates.html[latest updates] to prebuilt rules. To download the latest updates, refer to {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[Update Elastic prebuilt rules]. 
+To be added
```

Comment:
```suggestion
The redesigned alert details flyout gives you more options and flexibility when investigating an alert. Check the sections in the right panel to get a summary of important alert details. Analyze each section further by opening their detailed views. Additional core improvements include:

* Previews of rule details and visualizations allow you to stay within the flyout when investigating the alert.
* Investigation guides are easier to find and read.
* Alert insights now include prevalence information on related hosts and users.

```

Reply:
@jmikell821 you can reuse the [gif](https://www.elastic.co/guide/en/security/master/images/open-alert-details-flyout.gif) I created for the alert details docs. The file is `open-alert-details-flyout.gif` and it's located at: `docs/detections/images/open-alert-details-flyout.gif`

Reply:
Changes @paulewing suggested via Slack:

```suggestion
The redesigned alert details experience presents relevant context, insights, and more while investigating an alert. Utilize the collapsed view to see summarized information and then expand each section further to open detailed views. Additional improvements include:

* Previews of rule details and visualizations allow you to stay within the flyout when investigating the alert.
* Investigation guides are easier to find and read.
* Alert insights now include prevalence information on related hosts and users.
```